sdl_android/src/main/java/com/smartdevicelink/protocol/SdlPacket.java

```diff
@@ -65,6 +67,7 @@
 	int messageId;
 	int priorityCoefficient;
 	byte[] payload = null;
+	HashMap<String, Object> bsonPayload = new HashMap<>();
```

Comment:
We shouldn't instantiate this unless it is needed. The majority of packets won't use it so it's best to not allocate the memory.